bin/deploy

```diff
@@ -236,12 +236,19 @@ test -f "$LSSTSW/lsst_build/.deployed" || ( # Clone lsst_build
     touch "$LSSTSW/lsst_build/.deployed"
 )
 
-cat <<EOF
-
-Done. Run the following:
-
-    . $LSSTSW/bin/setup.sh
-
-to begin using it.
+echo "Done. Run the following:"
+echo 
+
+if [[ $SHELL =~ bash$ ]]; then
```

Comment:
Does this work? Won't `$SHELL` always be `bash` because this script is always run with `bash`? I don't think it's possible for this script to know what your login shell is. I think you just list all the options and let people choose. The options are enough to make people aware that they have a choice.